data-raw/amazon_books.R

```diff
@@ -0,0 +1,8 @@
+## code to prepare `amazon-books` dataset goes here
+library(tidyverse)
+
+amazon_books <- readr::read_csv("data-raw/amazon_books.csv")
+amazon_books <- amazon_books %>%
+  janitor::clean_names()
```

Comment:
You also don't need to call `janitor::` explicitly. Also please use this function piped to line 4